deluge/core/alertmanager.py

```diff
@@ -50,11 +50,10 @@ def __init__(self):
         # handlers is a dictionary of lists {"alert_type": [handler1,h2,..]}
         self.handlers = {}
         self.delayed_calls = []
-        self.wait_on_handler = False
 
     def update(self):
         self.delayed_calls = [dc for dc in self.delayed_calls if dc.active()]
-        self.handle_alerts(wait=self.wait_on_handler)
```

Comment:
We haven't discussed this before. The reason for adding the self.wait_on_handler variable in the first place was reduce the startup load time by avoiding processing all the torrent added events generated while adding the torrents to the session.
When using async_add instead, it's all asynchronous, so there is no longer the same need to postpone handling the events.